### PR TITLE
Add GitHub issue templates

### PR DESCRIPTION
**Pull request by Calinou**
Preview:

- [Bug report](https://github.com/Calinou/godot-vscode-plugin/blob/add-issue-templates/.github/ISSUE_TEMPLATE/bug_report.yml)
- [Feature request](https://github.com/Calinou/godot-vscode-plugin/blob/add-issue-templates/.github/ISSUE_TEMPLATE/feature_request.yml)

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.yml [hunk 1]
```diff
@@ -0,0 +1,69 @@
+name: Bug report
+description: Report a bug in Godot Tools VS Code add-on
+body:
```
Comment (raulsntos) [inline .github/ISSUE_TEMPLATE/bug_report.yml:3]: Might want to add `labels` like the [godot-csharp-vscode](https://github.com/godotengine/godot-csharp-vscode/blob/master/.github/ISSUE_TEMPLATE/bug-report.md) repository to automatically add the `bug` label to bug reports and `enhancement` to feature requests.

**Review by raulsntos — APPROVED** [review]

File: .github/ISSUE_TEMPLATE/feature_request.yml [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+    required: true
+
+- type: input
+  attributes:
+    label: System information
+    description: |
+      Specify the OS version, and when relevant hardware information.
+    placeholder: Windows 10
+  validations:
+    required: true
```
Comment (kurtlachmann) [inline .github/ISSUE_TEMPLATE/feature_request.yml:54]: I don't see how the used versions are relevant for a feature request?

Comment (Calinou) [reply .github/ISSUE_TEMPLATE/feature_request.yml:54]: The used versions can be useful to know the *context* in which the feature is being requested. I've seen numerous feature requests made by users on old versions, despite the feature being implemented in a newer version :slightly_smiling_face:

Comment (kurtlachmann) [reply .github/ISSUE_TEMPLATE/feature_request.yml:54]: I know what you mean. But the usual solution I'm aware of is putting a warning text at the top - something like "_Please make sure that this feature wasn't already requested or already implemented in the main branch._" With your template a user could still create a feature request and simply enter an outdated version. To me this form just feels like someone forgot to edit it after copy-pasting the bug report template. It's even talking about "reproducing an issue".

File: .github/ISSUE_TEMPLATE/feature_request.yml [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+  - enhancement
+body:
+
+- type: markdown
+  attributes:
+    value: |
+      - Write a descriptive issue title above.
+      - Search [open](https://github.com/godotengine/godot-vscode-plugin/issues) and [closed](https://github.com/godotengine/godot-vscode-plugin/issues?q=is%3Aissue+is%3Aclosed) issues to ensure it has not already been reported.
+      - Verify that you are using a [supported Godot version](https://docs.godotengine.org/en/stable/about/release_policy.html).
+      - This repository only contains the LSP *client*. Issues pertaining to the Godot editor itself or the LSP's *server* side should be reported on the [main Godot repository](https://github.com/godotengine/godot/issues).
```
Comment (Karmavil) [inline .github/ISSUE_TEMPLATE/feature_request.yml:13]: Not sure if ***issue*** is adequate to this context

**Review by Karmavil — APPROVED** [review]
Nice template. I added a comment to the feature request template for your consideration

File: .github/ISSUE_TEMPLATE/feature_request.yml (hunk 3 above)
Comment (Calinou) [reply .github/ISSUE_TEMPLATE/feature_request.yml:13]: A feature request is still an issue, technically :slightly_smiling_face:

**Review by mhilbrunner — APPROVED** [review]

**Pull request merged**